lib/v2/njucm/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'njucm.edu.cn': {
+        _name: '南京中医药大学',
+        lib: [
+            {
+                title: '博士招生',
```

Comment:
```suggestion
                title: '研究生院博士招生',
```
https://github.com/DIYgod/RSSHub/blob/d5d32b4c437c5844e4e7b689b2eb5da62d9ea559/docs/university.md?plain=1#L2249-L2251